src/config.js

```diff
@@ -0,0 +1,51 @@
+var fs = require("fs");
+var path = require("path");
+
+var PACKAGE_JSON = "package.json";
+
+var defaultConfig = {
+  testMatch: ["**/tests/**/*.js"]
```

Comment:
Gauge usually has properties loaded via a properties files which is available to use via environment variables. In this case, this property can be moved to js.properties.

Reply:
Do you want me to get rid of package.json load and replace it with js.properties load? Or try both sources?

Reply:
Try with js.properties load only.